mlrun/runtimes/sparkjob.py

```diff
@@ -161,10 +162,9 @@ def _run(self, runobj: RunObject, execution: MLClientCtx):
         if self.spec.image:
             update_in(job, "spec.image", self.spec.image)
         elif config.igz_version:
-            image_tag = (
-                "" if ":" in config.spark_app_image else ":" + config.igz_version
+            update_in(
+                job, "spec.image", config.spark_app_image + ":" + config.igz_version
```

Comment:
Why ?

Reply:
Now looking on what you did in Provazio I see why you did it,
but please still support the option to give image without tag

Reply:
done